website.py

```diff
@@ -37,6 +39,13 @@
         cmd.append("--allman")
     if "leftptr" in form:
         cmd.extend(["--pointer-style", "left"])
+    if "globals" in form:
+        cmd.append("--emit-globals")
+    if "visualize" in form:
+        cmd.append("--visualize")
+    function = form["functionselect"].value if "functionselect" in form else "all"
+    if function != "all":
+        cmd.extend(["--function", function])
```

Comment:
sanitize `function` to consist only of [a-zA-Z0-9_] similar to regvars

Reply:
I think `mips_to_c.py` is OK having garbage here? And we're not running it in a shell, so I don't think it's a security concern? 
(With regvars, it seems nice to strip `$` I guess?)

Reply:
Hm, yeah, maybe it's fine. There's a bit of an escaping hazard in that if the function name starts with `-` argparse will interpret it as an parameter, before giving up with an error about `--function` requiring an argument, and that sort of thing has made me cautious. Probably overly paranoid though, seems hard to imagine something actually dangerous coming from it :)